RELEASE_NOTE_DRAFT.md

```diff
@@ -44,6 +44,7 @@ The Eclipse Collections team gives a huge thank you to everyone who participated
 * Optimized asParallel() for immutable sets in O(1) instead of O(n).
 * Optimized some implementations of toString() that were delegating to iterators.
 * Improved equals() performance for Set implementations.
+* Allowed creating immutable primitive bags with billions of elements.
```

Comment:
You can roll this change back as we accumulate all of the commits into the release notes draft before release.

Reply:
Done